build.gradle.kts

```diff
@@ -21,6 +21,9 @@ val swagger: String by project
 val rsApi : String by project
 
 allprojects {
+    version = "1.0.0-SNAPSHOT"
+    group = "org.eclipse.dataspaceconnector.identityhub"
+
```

Comment:
nit: we can declare them as properties in gradle.properties, same as we do in EDC.

Also do we want version to be 0.0.1-SNAPSHOT initially instead?